### ISSUE TITLE
VoiceOver on Mac + Firefox fails to read MathML in PDF 2.0

### ISSUE THREAD

**Issue by teepeemm** [mozilla/for-firefox#88]
The site https://texlive.net/tests/MathML/ begins with six short files testing the capabilities of MathML in PDF 2.0.  NVDA (Windows) in Firefox reads these as

1. "heading level 1 1. heading level 1 Example. Some text. The cube root of x plus y."
2. "heading level 1 1. heading level 1 Example. Some text. The cube root of x plus y."
3. "heading level 1 1. heading level 1 Example. Some text. The cube root of x plus y."
4. "heading level 1 1. heading level 1 Example. Some text. LaTeX formula starts \begin {equation*} \sqrt [3]{x+y} \end {equation*} LaTeX formula ends the cube root of x plus y."  (The \\, {, }, [, and ] symbols are not pronounced, but NVDA records their presence in the speech viewer.)
5. "heading level 1 1 Example. Some text. The cube root of x plus y."
6. "heading level 1 1 Example. Some text. The cube root of x plus y."

VoiceOver (Mac) in Firefox reads these as

1. "heading level 1 1 Example, 2 items. 1 Example. Some text."
2. "heading level 1 1 Example, 2 items. 1 Example. Some text."
3. "heading level 1 1 Example, 2 items. 1 Example. Some text."
4. "heading level 1 1 Example, 2 items. 1 Example. Some text. LaTeX formula starts \begin {equation*} \sqrt [3]{x+y} \end {equation*} LaTeX formula ends."  (The [ and ] symbols are not pronounced, but VoiceOver records their presence in its captioning.)
5. "heading level 1 1 Example. Some text."
6. "heading level 1 1 Example. Some text."

The critical difference is that in cases 1, 2, 3, 5, and 6, the math is completely ignored.  Would Firefox on Mac be able to read the MathML?  My (ignorant?) assumption is that Firefox is sending the tagging information to VoiceOver.  Does this mean that Firefox doesn't see those tags?  Or does Firefox see the tags, but something goes wrong between Firefox and VoiceOver?
